### DISCUSSION TITLE
Windows taskbar not updating realtime after running Chris's util

### DISCUSSION THREAD

**Discussion by xodotLBO** [ChrisTitusTech/winutil#2866] [General]
Anyone know what could've caused this? When I say its not updating realtime I'm talking about discord, whatsapp, viber, telegram notifications and such. Before running it, when someone messages me on discord (an example) I get a number of messages (ex: 4) and after opening the chat it clears it in the taskbar.. but now, that number stays like that and updates maybe few minutes later or when you open and close the app (which is not ideal).

**Comment by zJvilian** [comment]
Having the same Issue over here, maybe the tweak set service to manual but haven't tried yet changing it
